### PR TITLE
[docs-only] Releasenotes for 1.10

### PR DESCRIPTION
**Pull request by dragotin**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by pmaier1** [comment]
??

https://github.com/owncloud/ocis/pull/2359

File: docs/ocis/release_notes.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ geekdocRepo: https://github.com/owncloud/ocis
 geekdocEditPath: edit/master/docs/ocis
 geekdocFilePath: release_notes.md
 ---
+## ownCloud Infinite Scale 1.10.0 Technology Preview
+
+Version 1.10.0 is another Tech Preview release of ownCloud Infinite Scale. More enhancements were added and the platform was more matured.
```
Comment (pascalwengerter) [inline docs/ocis/release_notes.md:11]:
```suggestion
Version 1.10.0 is another Tech Preview release of ownCloud Infinite Scale. More enhancements were added and the maturity of the platform has been improved.
```

**Review by pascalwengerter — COMMENTED** [review]
Almost good to go

**Comment by dragotin** [comment]
Closing in favour of the version of @pmaier1

**Pull request closed without merging**